src/MonoMod.Core/Platforms/PlatformTriple.cs

```diff
@@ -10,6 +10,7 @@
 using System.Linq;
 using System.Reflection;
 using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;

```

Comment:
Lets not modify files that don't need to be :)

Reply:
left over